packages/theme/src/theme/_forms.scss

```diff
@@ -333,9 +359,36 @@ form {
 	}
 
 	.form-group-lg {
-		select,
-		select.form-control {
-			line-height: 1.5;
+		select {
+			&.form-control {
+				line-height: 1.5;
+			}
+		}
+	}
+
+	.form-group > .form-control::placeholder {
+		color: $silver;
+	}
+
+	.form-control + label {
+		font-size: $font-size-base;
+		color: $silver-chalice;
+	}
+
+	textarea {
+		&.form-control {
```

Comment:
`textarea { &.form-control {} }` --> `textarea.form-control {}`

Reply:
tag.className (select.form-control) fails with lint:style checks
"error    Qualifying elements are not allowed for class selectors  no-qualifying-elements"
so we are using "tag {  &.className { .......... } }"